superset-frontend/src/SqlLab/components/RunQueryActionButton/index.tsx

```diff
@@ -114,7 +117,9 @@ const RunQueryActionButton = ({
         tooltip={
           (!isDisabled &&
             (shouldShowStopBtn
-              ? t('Stop running (Ctrl + x)')
+              ? userOS === 'MacOS'
```

Comment:
we could move this to a memo, or even a plain function outside of the component called `getStopButtonTooltipText` or something.. Seems too much inline ifs for my liking here

Reply:
Willing to take a look at externalizing this, and squeeze it into this PR, @codemaster08240328?